automation/exception_context.py

```
@@ -0,0 +1,68 @@
+    pass
+
+
+def _is_browser_closed_error(e: BaseException) -> bool:
+    """检查是否为浏览器关闭相关的错误"""
+    if isinstance(e, BrowserClosedError):
+        return True
+    # 检查异常链中是否包含浏览器关闭错误
+    if e.__cause__ and _is_browser_closed_error(e.__cause__):
+        return True
+    # 检查错误消息
+    # 不依赖 Playwright 的内部异常类型；用消息匹配即可覆盖“目标已关闭”等场景。
```

Ruff (0.14.14)

Comment contains ambiguous `；` (FULLWIDTH SEMICOLON). Did you mean `;` (SEMICOLON)?

(RUF003)